dfastmi/gui/dialog_model.py

```diff
@@ -221,12 +223,9 @@ def run_analysis(self) -> bool:
             We call batch_mode_core which can throw and log an exception.
             If thrown, analysis has failed.
         """
-        try:
-            success = dfastmi.batch.core.batch_mode_core(
-                self.rivers, False, self.config
-            )
-        except:
-            success = False
+        success = dfastmi.batch.core.batch_mode_core(
```

Comment:
We always, provide default = false for GUI parameter, no need to pass through

Reply:
or we need to provide always **True** here...

Reply:
See comment above.

Reply:
See PR #103, Discussed with Ralph.